src/controller_examples/zookeeper_controller/spec/zookeeper_api.rs

```diff
@@ -120,6 +122,39 @@ impl ExternalAPI for ZKAPI {
     }
 }
 
+pub open spec fn validate_config_map(name: StringView, namespace: StringView, resources: StoredState) -> bool {
+    let cm_key = ObjectRef {
+        kind: Kind::ConfigMapKind,
+        namespace: namespace,
+        name: name + new_strlit("-configmap")@,
+    };
+    let cm_data = ConfigMapView::unmarshal(resources[cm_key]).get_Ok_0().data.get_Some_0();
+    &&& resources.contains_key(cm_key)
+    &&& ConfigMapView::unmarshal(resources[cm_key]).is_Ok()
+    &&& ConfigMapView::unmarshal(resources[cm_key]).get_Ok_0().data.is_Some()
+    &&& cm_data.contains_key(new_strlit("zoo.cfg")@)
+    &&& forall |min_i, min_j, max_i, max_j, min, max|
+            cm_data[new_strlit("zoo.cfg")@].subrange(min_i,min_j) == new_strlit("minSessionTimeout=")@ + int_to_string_view(min)
+            && cm_data[new_strlit("zoo.cfg")@].subrange(max_i,max_j) == new_strlit("maxSessionTimeout=")@ + int_to_string_view(max)
+            ==> min <= max
```

Comment:
#405 also introduces a validation rule for non-zero timeout, can you also add that here?

Reply:
Yes, added